docs/ExportPyTorchCustomOps.md

```diff
@@ -0,0 +1,173 @@
+
+## Export of Custom Ops for ONNX Runtime
+
+This document explains the process of exporting PyTorch models with custom ONNX Runtime ops.
+The aim is to export a PyTorch model with operators that are not supported in ONNX, and extend ONNX Runtime to support these custom ops.
+
+Currently, a torch op can be exported as a custom operator using our custom op (symbolic) registration API. We can 
+use this API to register custom ONNX Runtime ops under "com.microsoft" domain.
+
+
+### 1. Exporting a Custom Op
+
+In this example, we take Inverse operator as an example. To enable export of ```torch.inverse```, a symbolic function 
+can be created and registered as part of custom ops:
+
+```python
+from torch.onnx import register_custom_op_symbolic
+
+def my_inverse(g, self):
+    return g.op("com.microsoft::Inverse", self)
+
+# register_custom_op_symbolic('<namespace>::inverse', my_inverse, <opset_version>)
```

Comment:
maybe explain what the <namespace> here is referring to. From the below example, can we say that namespace can be omitted for official torch operators?